### DISCUSSION TITLE
why shold cvsection be longer than 3 character

### DISCUSSION THREAD

**Discussion by xiaoweing** [posquit0/Awesome-CV#471] [Q&A] [answered]
in awesome-cv.cls 

>  \def\@sectioncolor#1#2#3{%
  \ifbool{acvSectionColorHighlight}{{\color{awesome}#1#2#3}}{#1#2#3}%
}
% Define a section for CV
% Usage: \cvsection{<section-title>}
\newcommand{\cvsection}[1]{%
  \vspace{\acvSectionTopSkip}
  \sectionstyle{#1}
  \phantomsection
  \color{sectiondivider}\vhrulefill{0.9pt}
}
\newcommand*{\sectionstyle}[1]{{\fontsize{16pt}{1em}\bodyfont\bfseries\color{text}\@sectioncolor #1}}

obviously, the section title shoulld be longer than 3 character, or latex will compile error

> Paragraph ended before \@sectioncolor was complete. \cvsection{IT}

i don't know why, can this be more flexible? locally i do this to avoid

>  \def\@sectioncolor#1{%
  \ifbool{acvSectionColorHighlight}{{\color{awesome}#1}}{#1}%
}

**Comment by OJFord** [comment, marked as answer]
It's because the first three letters are coloured differently, see the example in the readme.

PR welcome for lifting the limitation though, as long as it continues to work the same way for headings >=3 characters.

**Reply by xiaoweing** [reply]
thanks, it make sense. I got this question because i want to add a cvsection whose title is "IT"
